src/chrome/content/rules/Nstatic.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Nstatic.net">
+
+	<target host="nstatic.net" />
+
+	<rule from="^http:" to="https:" />
+
+</ruleset>
```

Comment:
Are there any subdomains?

Reply:
I'd rather keep the changes in this PR as minimal as possible. Only run hsts-prune and fix the result. Enhancements can take place in future PRs

Reply:
I will make such PR after this is merged.

Reply:
merged.